test/packer_basic_example_test.go

```diff
@@ -51,6 +52,43 @@ func TestPackerBasicExample(t *testing.T) {
 	assert.True(t, amiIsPublic)
 }
 
+func TestPackerMultipleConcurrentAmis(t *testing.T) {
+	t.Parallel()
+
+	// Build a map of 3 randomId <-> packer.Options, in 3 random AWS Regions
+	// then build all of these AMIs in parallel and make sure that there are
+	// no errors.
+	var identifierToOptions = map[string]*packer.Options{}
+	for i := 0; i < 3; i++ {
+		// Pick a random AWS region to test in. This helps ensure your code works in all regions.
+		awsRegion := terratest_aws.GetRandomRegion(t, nil, nil)
+
+		packerOptions := &packer.Options{
+			// The path to where the Packer template is located
+			Template: "../examples/packer-basic-example/build.json",
+
+			// Variables to pass to our Packer build using -var options
+			Vars: map[string]string{
+				"aws_region":    awsRegion,
+				"ami_base_name": random.UniqueId(),
```

Comment:
Include some reasonable name as well as `UniqueId` so when debugging, we know where these images are coming from.

Reply:
done